src/main/scala/nl/biopet/tools/findoverlapmatch/FindOverlapMatch.scala

```diff
@@ -10,8 +15,92 @@ object FindOverlapMatch extends ToolCommand {
 
     logger.info("Start")
 
-    //TODO: Execute code
+    val reader = Source.fromFile(cmdArgs.inputMetrics)
+
+    val data = reader.getLines().map(_.split("\t")).toArray
+
+    val samplesColumnHeader = data.head.zipWithIndex.tail
+    val samplesRowHeader = data.map(_.head).zipWithIndex.tail
+
+    var overlap = 0
+    var multiOverlap = 0
+    var noOverlap = 0
+    var correctMatches = 0
+    var incorrectMatches = 0
+
+    val writer = cmdArgs.outputFile match {
+      case Some(file) => new PrintStream(file)
+      case _ => sys.process.stdout
+    }
+
+    val matchesRegexes = cmdArgs.shouldMatchRegexFile.map { file =>
```

Comment:
Did you mean matchedRegexes?

Reply:
that sound for me that that the regexes are matched while they are used for matching